service/matching/db.go

```diff
@@ -222,7 +222,7 @@ func (db *taskQueueDB) updateApproximateBacklogCount(
 	// Prevent under-counting
 	if db.approximateBacklogCount.Load()+delta < 0 {
 		db.logger.Info("ApproximateBacklogCounter could have under-counted.",
-			tag.WorkerBuildId(db.queue.BuildId()), tag.WorkflowNamespace(db.queue.Partition().NamespaceId()))
+			tag.WorkerBuildId(db.queue.Version().MetricsTagValue()), tag.WorkflowNamespace(db.queue.Partition().NamespaceId()))
```

Comment:
shouldn't this be
```suggestion
			tag.WorkerBuildId(db.queue.Version().MetricsTagValue()), tag.WorkflowNamespaceId(db.queue.Partition().NamespaceId()))
```
? mixing up name/id